modules/integration_azure-mariadb/conf/01-cpu.yaml

```diff
@@ -0,0 +1,22 @@
+module: Azure MariaDB
+name: "CPU usage"
```

Comment:
defining id like
`id: cpu`
will make variables naming shorter and "usage" is not really useful here.
can be applied to other detectors if you want.